iocore/net/UnixUDPConnection.cc

```diff
@@ -116,7 +116,7 @@ UDPConnection::bindToThread(Continuation *c)
   AddRef();
   uc->continuation = c;
   mutex            = c->mutex;
-  ink_atomiclist_push(&get_UDPNetHandler(t)->udpNewConnections, uc);
```

Comment:
The new udp connection is pushed into the atomic list.
The `udpNewConnections` is redefined with `ASLL` macro.